ports/g3log/portfile.cmake

```
@@ -28,7 +28,7 @@ vcpkg_install_cmake()
 
 vcpkg_copy_pdbs()
 
-vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/g3logger TARGET_PATH share/g3logger)
+vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/g3log TARGET_PATH share/g3log)

```

```suggestion
vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/g3log)
```
`TARGET_PATH  `will be specified to `share/${PORT}` by default. So it's unnecessary to add this argument.